test/bin/windows-shims.js

```diff
@@ -44,16 +44,14 @@ t.test('shim contents', t => {
 
   t.test('bash', t => {
     const { diff, letters } = diffFiles(SHIMS.npm, SHIMS.npx)
-    t.match(diff[0].split('\n').reverse().join(''), /^NPX_CLI_JS=/, 'has NPX_CLI')
-    t.equal(diff.length, 1)
+    t.equal(diff.length, 0)
     t.strictSame([...letters], ['M', 'X'], 'all other changes are m->x')
     t.end()
   })
 
   t.test('cmd', t => {
     const { diff, letters } = diffFiles(SHIMS['npm.cmd'], SHIMS['npx.cmd'])
-    t.match(diff[0], /^SET "NPX_CLI_JS=/, 'has NPX_CLI')
-    t.equal(diff.length, 1)
+    t.equal(diff.length, 0)
```

Comment:
```suggestion
    t.equal(diff, [])
```

This way a test failure will show what's in there instead of a less helpful "length doesn't match" error>

Reply:
There's another one for pwsh

Reply:
👍🏼 i think this might need to be `t.strictSame` though?

Reply:
Yeah I'll have to do it locally since there's a diff to be made that's not part of this PR